### ISSUE TITLE
Toolbar: Missing a11y Toolbar Pattern

### ISSUE THREAD

**Issue by Abhi1845** [SAP/ui5-webcomponents-react#8389]
### Describe the bug

Customer is expecting Footer to follow [ARIA Toolbar Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/) likewise it is working in UI5 https://ui5.sap.com/#/entity/sap.m.Bar/sample/sap.m.sample.Page/code

### Reproduction steps

1. Open https://ui5.github.io/webcomponents-react/v2/?path=/docs/layouts-floorplans-toolbar--docs
2. Try as instructed in  [ARIA Toolbar Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/)
...

### UI5 Web Components for React Version

"2.18.1"

### UI5 Web Components Version

"2.18.1"

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Hi @Abhi1845 

I will close this issue as there is already [another one](https://github.com/UI5/webcomponents/issues/12945) covering this topic. If you have any questions or more information, please address them in the original issue.
